website/docs/whats-new/release-notes/partials/design-tokens.md

```diff
@@ -0,0 +1,82 @@
+ **`npm package: @hashicorp/design-system-tokens`**
+
+## 1.8.0
+
+**Month Day, Year**
+
+- `Design tokens` added color tokens for “Vault Secrets” product. 
+
+- `IconTile` updated component to include `vault-secrets` product option.
+
+[(#1640)](https://github.com/hashicorp/design-system/pull/1640) Thanks [@didoo](https://github.com/didoo)!
+
+## 1.7.0
+
+**Month Day, Year**
+
+- `marketing` added JSON output format for target (and in the process refactored internal logic for tokens generation). [(#1513)](https://github.com/hashicorp/design-system/pull/1513) Thanks [@dstaley](https://github.com/dstaley)! 
```

Comment:
Token entries are different than component entries, where there may not be a good element to list at the beginning. We should aim for clarity vs trying to align too much to one specific format. I'll add a note about this in the instructions doc.

```suggestion
- Added JSON output format for `marketing` target and refactored internal logic for token generation. [#1513](https://github.com/hashicorp/design-system/pull/1513) Thanks [@dstaley](https://github.com/dstaley)!
```